falcon/request.py

```diff
@@ -355,32 +355,34 @@ class Request(object):
         range_unit (str): Unit of the range parsed from the value of the
             Range header, or ``None`` if the header is missing
         if_match (list): Value of the If-Match header, as a parsed list of
-            :class:`falcon.ETag` objects or ``None`` if the header is missing.
+            :class:`falcon.ETag` objects or ``None`` if the header is missing
+            or its value is blank.
 
-            This property is determined by the value of ``HTTP_IF_MATCH``
-            in the WSGI environment dict.
+            The header is parsed on-demand, and so if you need to reuse the
+            result it is best to assign it to a variable rather than reference
```

Comment:
I am quote hopeless at English punctuation, so this is just a flag for you to check if you intentionally meant it this way -- I would put a comma after _result_.

Reply:
Due to the clauses being joined by *and*, placing a comma there would make it a little choppier than I would like. However, the sentence in its current form is admittedly rather long. It may be best to simply split it into two sentences.